src/proto/grpc/testing/control.proto

```diff
@@ -214,6 +214,8 @@ message Scenarios {
 // once the scenario has finished.
 message ScenarioResultSummary
 {
+  // Requested QPS for fixed load tests.
+  double attempted_qps = 19;
```

Comment:
I think we seem to sort proto by tag, not alphabetically, wdyt?

Reply:
Done